baseapp/abci_utils.go

```diff
@@ -125,12 +107,20 @@ func ValidateVoteExtensions(
 			return fmt.Errorf("failed to verify validator %X vote extension signature", valConsAddr)
 		}
 
-		sumVP = sumVP.Add(validator.BondedTokens())
+		bondedTokens, err := valStore.BondedTokensByConsAddr(ctx, valConsAddr)
```

Comment:
Curious, isn't it slower now because we get the validator twice? While before only once.
It is probably negligible but just curious. :D

Reply:
Yes, it will be slower sadly. We could have a specific API that returns both bonded tokens and public key, wdyt?

Reply:
No objection to that personally.

Reply:
@facundomedica yes let's do that. This will be used in the VE hotpath so we don't want obvious performance slowdowns

Reply:
Added `BondedTokensAndPubKeyByConsAddr`